rpc/legacyrpc/methods.go

```diff
@@ -2681,7 +2682,24 @@ func setTicketVoteBits(icmd interface{}, w *wallet.Wallet) (interface{}, error)
 		return nil, err
 	}
 
-	err = w.SetVoteBitsForTicket(ticket, cmd.VoteBits)
+	var voteBitsExt []byte
+	if cmd.VoteBitsExt != nil {
+		voteBitsExt, err = hex.DecodeString(*cmd.VoteBitsExt)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(voteBitsExt) > stake.MaxSingleBytePushLength-2 {
+			return nil, fmt.Errorf("bad ext votebits length (got %v, max %v)",
```

Comment:
Why do this check here instead of relying on `stake.SetVoteBitsForTicket` erroring?